themes/socialbase/templates/block/block--system-menu-block--main.html.twig

```diff
@@ -49,7 +49,7 @@
   {% if content.links.search_block is defined %}
     <div class="navbar__open-search-control">
       <a href="{{ '/search/all' }}" class="navbar__open-search-block" rel="search" title="{{ 'Click to open search box'|t }}">
-        <span class="invisible">{{ 'Search'|t }}</span>
+        <span class = "sr-only" aria-hidden="true">{{ 'Search'|t }}</span>
```

Comment:
@agami4 Why did you add the `aria-hidden="true"`? This way it's still not visible to screenreaders.

Reply:
@Kingdutch, I've removed the `aria-hidden` attribute and checked via 'Voice Over'